.circleci/config.yml

```diff
@@ -59,71 +71,161 @@ commands:
         type: string
     steps:
       - checkout
+      - restore_cache:
+          key: -<< pipeline.parameters.cache_key_version >>-win-sys-{{ .Branch }}-<< parameters.py_version >>
       - run:
-          name: Preparing system
+          name: "Preparing environment - system"
```

Comment:
You don't normally need to quote strings in YAML. this can be unquoted.

Reply:
good to know. Removed all of them.